src/AudioManagerImpl.js

```diff
@@ -232,6 +232,7 @@ export default class AudioManagerImpl implements AudioManager {
                 window.speechSynthesis.cancel();
             }
             this.playStringMessage(message);
```

Comment:
Why wouldn't you set the flag in `playStringMessage`?  In other words, are there any circumstances in which you'd play feedback without setting the flag?

Reply:
In reading the code, it seems like the intent of the flag is to avoid event attempting to play a preview if feedback is already being announced?  Is this correct?

Reply:
No, the intention is to prevent preview announcement played by focus when interacting with components, if there is a feedback announcement. Without this flag, feedback announcement about program step added and deleted is not played because of the preview announcement. While feedback announcement is playing, another feedback announcement can stop that and played.

Reply:
Then it definitely still needs work, as you could trivially clear the flag by changing focus rapidly. See my other note for a suggested fix.